src/__tests__/calculate.test.js

```diff
@@ -0,0 +1,33 @@
+import calculate from '../logic/calculate';
```

Comment:
- [ ] Hi @Salimer, it seems we are missing the scenario when the button name is "**=**" and "**0**". Let's kindly address the issue and submit our code once more.